packages/eligibility/src/Eligibility.js

```diff
@@ -101,20 +102,29 @@ function Eligibility() {
     let i = 0;
     let scoreValue = 0;
     let questionsList = [];
+    let maybeList = [];
+
     while (i < 9) {
-      if(answersList[i] === "Yes") {
+      if((i===3 || i===5) && answersList[i] === "No") {
```

Comment:
@rachita19082: while the code is technically correct, I would suggest to encode this information in the [api/quizQuestions.js](https://github.com/unicef/publicgoods-scripts/blob/master/packages/eligibility/src/api/quizQuestions.js) as you had before when you encoded the `yes` expected answer if I recall correctly, so that we have all the information related to the questions in one place, and we can separate the code from the data.

Reply:
Resolved in commit 69658e5.

Reply:
@lacabra Please let me know if this looks fine.